src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py

```diff
@@ -0,0 +1,20 @@
+def reset_entity_type(doc):
+    '''
+    In the "Dataset" search results, we don't want to show "Datasets"
+    which are only there to provide visualization.
+    As vis-lifting is implemented in portal-ui, we will match
+    the corresponding entities here, and reset the entity_type.
+
+    >>> doc = {
+    ...     'data_types': ['image_pyramid'],
+    ...     'entity_type': 'Dataset'
+    ... }
+    >>> reset_entity_type(doc)
+    >>> doc['entity_type']
+    'Support'
```

Comment:
Is the idea we would not display `Support` types?

Reply:
yes... old urls would still work, but we don't want to show them in the list of search results.